src/actions/app.js

```diff
@@ -41,6 +41,16 @@ export function setProfileSharingStatus(
   };
 }
```

Comment:
I think we have generic problems around the handling of the back button before loading a profile, and that's not generic to this.

For example you can load any profile (eg https://perf-html.io/public/8c8e1c9f7feb95c69e1eee7b059d4ac33eb9194f/calltree/?globalTrackOrder=0-1-2-3-4-5-6-7-8-9-10&hiddenGlobalTracks=1-2-3-4-5-6-7-8-9-10&localTrackOrderByPid=11808-0~&react_perf&thread=0&v=3), then press back, and you'll get an error as well. (but not the same one)

Here the problem is that we use selectors that need a selectedThreadIndex, but pressing "back" gets us a state that doesn't have this yet, and then we get the error. I'd rather not tackle this problem in this PR because I think this has a broad scope unfortunately. What do you think?

Reply:
I filed https://github.com/devtools-html/perf.html/issues/1653